acceptance-tests/build.gradle

```diff
@@ -90,15 +90,15 @@ dependencies {
 test.enabled = false
 
 task cleanBesuInstall(type: Delete) {
-  delete "$buildDir/besu"
+  delete layout.buildDirectory.dir("besu")
 }
 
 task downloadBesu(type: Download) {
   src {
     // see gradle.properties for Besu URL
     return new SimpleTemplateEngine().createTemplate(besuDistroUrl).make(["besuVersion":"$besuVersion"]).toString()
   }
-  dest new File(buildDir, "besu-${besuVersion}.tar.gz")
+  dest layout.buildDirectory.file("downloads/besu-${besuVersion}.tar.gz")
```

Comment:
Does the downloads directory get created if doesn't exist? And will work on Windows with the hardcoded `/` directory separator?

Reply:
This works on nix without any problems. Need to check our whole build on Windows environment.